### ISSUE TITLE
New SyntaxError case 

### ISSUE THREAD

**Issue by aroberge** [friendly-traceback/friendly-traceback#70]
See https://bugs.python.org/issue45727

```
[1]: 1 2 3 4

  Code block [1], line 1
    1 2 3 4
    ^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?

        Friendly-traceback does not know the cause of this error.

```

**Comment by aroberge** [comment]
The message is the same with 3.10.  Prior versions only showed SyntaxError.

These examples cannot be fixed using the suggestion from CPython (3.10, 3.11). Since there is not a single typo to fix, we will not be able to give a constructive suggestion - but could point out that CPython's suggestion does not fix the problem.

**Comment by aroberge** [comment]
I've decided against making such a suggestion.
